pyiron_base/generic/hdfio.py

```diff
@@ -291,40 +291,50 @@ def copy_to(self, destination, file_name=None, maintain_name=True):
             FileHDFio: FileHDFio object pointing to a file which now contains the same content as file of the current
                        FileHDFio object.
         """
+        def _internal_copy(source, target, dest, h5_path):
+            if dest.h5_path[0] == "/":
+                dest_path = dest.h5_path[1:]
+            else:
+                dest_path = dest.h5_path
+            if maintain_name:
+                try:
+                    target.create_group(dest_path)
+                except ValueError:
+                    pass
```

Comment:
Can we get a quick in-line comment about what we're passing over/why it's fine?

Reply:
Done